src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpEnvironmentProxy.cs

```diff
@@ -135,8 +138,8 @@ public static bool TryCreate(out IWebProxy proxy)
                 proxy = null;
                 return false;
             }
-
-            proxy = new HttpEnvironmentProxy(httpProxy, httpsProxy, Environment.GetEnvironmentVariable(EnvNoProxyLC));
+            string noProxy = Environment.GetEnvironmentVariable(EnvNoProxyLC) ?? Environment.GetEnvironmentVariable(EnvNoProxyUC);
```

Comment:
Nit: please add a blank line above this

Reply:
I will add it in next commit with tests also fixed.